ansible_collections/arista/cvp/molecule/cv_container_v3/converge.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Test cv_container_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: false
+  collections:
+    - arista.cvp
+  vars:
+    CONTAINERS_PROVISION_ADD_IMAGE:
+        Test_container:
+            configlets: []
+            imageBundle: EOS-4.25.4M
+            parentContainerName: Tenant
+
+  tasks:
+    - name: "Build Container topology on {{inventory_hostname}}"
```

Comment:
Can we add tests for the following?
1. Create a 3 tier topology
2. Delete the above topology
3. Create a 3 tier topology and attach configlet to one of the containers and remove it
4. Create a 3 tier topology and attach image bundles to one of the containers and remove it
5. Play around with `apply_mode: loose` and `apply_mode: strict`. 
Refer to [cv_container_v3](https://cvp.avd.sh/en/stable/docs/how-to/v3/cv_container_v3/) for more info on how `apply_mode` works.
NOTE: `apply_mode` works for both configlets and imageBundle, so we need to test both

Reply:
Fixed!